_includes/resource.html

```diff
@@ -5,7 +5,9 @@ <h4><a href="{{ resource.href }}">{{ resource.title }}</a></h4>
 
     <p class="phones">
         {% if resource.phones.primary %}
-            <a href="tel:{{ resource.phones.primary.link }}"><span class="phone-icon" role="img" aria-label="Phone"></span></a>
+            <a href="tel:{{ resource.phones.primary.link }}"><span class="phone-icon" role="img" aria-label="Phone">
+                <svg xmlns="http://www.w3.org/2000/svg" class="phone-light" width="24" height="24" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-phone-call"><path d="M15.05 5A5 5 0 0 1 19 8.95M15.05 1A9 9 0 0 1 23 8.94m-1 7.98v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z"></path></svg>
```

Comment:
To avoid repeating the same glyph all over the page I wonder if we could define it once and then use the [`use`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use) element to reference it? I believe it'll still be styleable.

I've never tried this before though, but it seems wrong to repeat the glyph over and over again on the page.

Reply:
Do you think we should just define it once as a hidden SVG and then just reference the definition on each resource? I'm not sure how we'd do it easily otherwise

Reply:
Yeah that'd be the idea I think.

Reply:
Although I wonder if we could just have `phone-light` and `phone-dark` and then use CSS selectors to toggle one or the other as `display: none` depending on the selected color scheme? Then we can render both as `img src="/whatever"` instead of inlining the SVG.

Reply:
Oh that makes sense! I can play around with that - do you mind if I do that in a follow up PR though? I'd love to get this merged & the repo transferred and I don't want it contingent on my ability to mess with the proper display modes :sweat_smile: